setup.sh

```diff
@@ -1,56 +1,64 @@
 #!/bin/bash
-time=`date '+%Y%m%d_%H%M'`
-VAG_COMMON_DIR="/vagrant/"
+time=`TZ=Asia/Jerusalem date '+%Y%m%d_%H%M'`
+VAG_COMMON_DIR="/vagrant"
+HOME="/home/vagrant"
 LOG_DIR="$VAG_COMMON_DIR/log"
 OUTFILE="$LOG_DIR/${time}_vagrant-out.log"
+FLASK_APP_PY="$VAG_COMMON_DIR/flask_init.py"
+FLASK_DIR="$HOME/FlaskApp"
+FLASK_EXEC="$HOME/runFlask.sh"
+KILLFLASK_SCRIPT="$VAG_COMMON_DIR/killFlask.sh"
+PYTHON3="`which python3`"
```

Comment:
executing "python3" will always give you the value returned by "which python3".
If the only think you are using this for is to execute python, just use the python3 executable.

Reply:
Noted. Will be applied in next commit.